tools/cli/src/webpack/config.ts

```diff
@@ -404,6 +398,11 @@ export const createConfiguration: (
         maxInitialRequests: Infinity,
         chunks: 'all',
         cacheGroups: {
+          errorHandler: {
+            test: /global-error-handler/,
```

Comment:
IMO explicit entry is better for maintainance. affine devs often do refactors that the rule here might be missed in the future

Reply:
we need take care more if choosing multi entries

Reply:
you can try doing it if it helps